fetch-utils.js

```diff
@@ -42,6 +66,11 @@ export async function updateChatchphrases(value){
     const currentUserId = client.auth.user().id;
 
     // in supabase, update the catchphrases property
+    const response = await client 
+        .from('characters')
+        .update({ catchphrases: value })
```

Comment:
It is possible there is something going on with the way you are destructuring here that is causing your catchphrases to not update properly.